addons/mail/models/mail_template.py

```diff
@@ -238,6 +242,28 @@ def onchange_sub_model_object_value_field(self):
             self.null_value = False
 
     @api.multi
+    def write(self, vals):
+        # This override is for checking that the template is correct when
+        # saving it
+        test_model = 'res.partner'
+        Model = self.env[test_model]
+        # Select a random record to test if rendering works
+        res_ids = Model.search_read([], ['id'], limit=100)
+        res_ids = [res['id'] for res in res_ids]
```

Comment:
this part should be inside the `if vals.get('body_html'):`

Reply:
You're right, thanks